app/components/grid.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import clsx from 'clsx'
+import type {ReactNode} from 'react'
+
+interface GridProps {
+  children: ReactNode
+  overflow?: boolean
+  className?: string
+}
+
+function Grid({children, className}: GridProps) {
+  return (
+    <div
+      className={clsx(
+        'gap-[18px] mx-[10vw] md:gap-[17px] lg:gap-[23px] grid grid-cols-4 md:grid-cols-8 lg:grid-cols-12',
```

Comment:
I can't remember what this syntax is called, but if we can avoid it let's do in effort to keep sizes consistent. If it's a matter of the design being off from one of the pre-set sizes by a pixel or two then let's just go with the pre-set size rather than using this syntax. I'm concerned that overusing this reduces consistency.

Reply:
I agree. I'll change it.

Regarding the grid. I think we should add a max-width. Things get weird when making the browser 3000px wide for example.